mlrun/frameworks/ml_base/mlrun_interface.py

```diff
@@ -0,0 +1,59 @@
+import pandas as pd
+from mlrun.frameworks._common.plots import eval_model_v2
+from mlrun.frameworks._common import MLRunInterface
+from cloudpickle import dumps
+
+class MLBaseMLRunInterface(MLRunInterface):
+    """
+    Wraps the original .fit() method of the passed model enabling auto-logging.
+    """
+
+    @classmethod
+    def add_interface(cls, model, context, data, *args, **kwargs):
+        """
+        Wrap the given model with MLRun model features, providing it with MLRun model attributes including its
+        parameters and methods.
+        :param model: The model to wrap.
+        :param context: MLRun context to work with. If no context is given it will be retrieved via
+                        'mlrun.get_or_create_ctx(None)'
+        :param data: The train_test_split X_train, X_test, y_train, y_test.
```

Comment:
you can take the x_train, y_train from the fit method kwargs. remove data and add x_test, y_test instead.